.env

```diff
@@ -6,4 +6,4 @@ ADDRESS_TO=
 DISABLE_CONFIRMATION=false
 PRIVATE_KEY=
 FAUCET_QUOTIENT=10
-FAUCET_URL=https://proxy.teststand.neontest.xyz/request_erc20_tokens
+FAUCET_URL=https://proxy.teststand.neontest.xyz/request_erc20_tokens
```

Comment:
Just curious, why did you remove the ending empty line? It's like a standard in many styles to have an empty line in the end of file